### DISCUSSION TITLE
Welcome to Mango Ingest Discussions!

### DISCUSSION THREAD

**Discussion by paulborgermans** [kuleuven/mango-ingest#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by u0046038** [comment]
Kan deze tool ook geïnstalleerd worden op een Windows systeem ?
Zo ja, kan je me hiervoor de instructies bezorgen ?
